css-two-column-layout/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  max-width: 1050px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  letter-spacing: 0.1rem;
+  line-height: 1.5rem;
+}
+
+.column-one-forth {
```

Comment:
make sure your column classes have actual width set on them. Right now flexbox is controlling the column size and you don't want that :)